app/processing/siti-agri/handle-known-defects/cap-bps-penalties/index.js

```diff
@@ -0,0 +1,16 @@
+const { P02, P04 } = require('../../../../constants/line-descriptions')
+const { recalculateBPSPenalties } = require('../cap-bps-penalties/recalculate-bps-penalties')
+const { bps } = require('../../../../schemes')
+
+const capBPSPenalties = (paymentRequest) => {
+  const penaltyInvoiceLines = paymentRequest.invoiceLines.filter(invoiceLine => invoiceLine.description === P02 || invoiceLine.description === P04)
```

Comment:
This no longer works as line descriptions don't equal the amended constant values.  Needs to be starts with these codes rather than is these codes